src/main/java/eu/enmeshed/model/request/requestItems/RequestItemGroup.java

```diff
@@ -15,8 +15,8 @@
 @SuperBuilder
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "@type")
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
-public class RequestItemGroup {
-  private List<RequestItem> items;
+public class RequestItemGroup extends RequestItem {
+  private List<RequestItemDerivation> items;
   private String title;
```

Comment:
`title`, `description` and `metadata` can be removed here, because `RequestItemGroup extends RequestItem`, right?